### PR TITLE
Update Custom Rules documentation

### PR DESCRIPTION
**Pull request by pedrosousa**
* Custom Firewall rules are now available in the dashboard.
* Rate Limiting phase is now `http_ratelimit`.

Addresses https://jira.cfops.it/browse/PCX-1749.

### CODE REVIEW

**Review by rboulton-cf — APPROVED** [review]
Looked through the changes, and am happy they're accurate. They also seem clear to me.

File: products/firewall/src/content/cf-rulesets/custom-rules/custom-firewall/create-api.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+pcx-content-type: how-to
+order: 2
+type: overview
+---
+
+# Create Custom Firewall rules via API
+
+<Aside type='warning' header='Important'>
```
Comment (kodster28) [inline products/firewall/src/content/cf-rulesets/custom-rules/custom-firewall/create-api.md:13]: Might want to put this at below the `You must deploy...` for SEO description.

File: products/firewall/src/content/cf-rulesets/custom-rules/custom-firewall/create-api.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+Use the [Rulesets API](/cf-rulesets/rulesets-api) to create a Custom Firewall rule via API.
+
+You must deploy Custom Firewall rules to the `http_request_firewall_custom` phase ruleset.
+
+## Create a Custom Firewall rule
+
+To create a Custom Firewall rule, add a rule to the `http_request_firewall_custom` phase ruleset.
+
+Do the following:
```
Comment (kodster28) [inline products/firewall/src/content/cf-rulesets/custom-rules/custom-firewall/create-api.md]: Feel like you could just remove this line.

File: products/firewall/src/content/cf-rulesets/custom-rules/custom-firewall/create-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+pcx-content-type: how-to
+order: 1
+type: overview
+---
+
+# Create Custom Firewall rules in the dashboard
+
+<Aside type='warning' header='Important'>
```
Comment (kodster28) [inline products/firewall/src/content/cf-rulesets/custom-rules/custom-firewall/create-dashboard.md:11]: Same comment as above about the `Aside`

File: products/firewall/src/content/cf-rulesets/custom-rules/custom-firewall/index.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+pcx-content-type: concept
+order: 2
+---
+
+# Custom Firewall rules
+
+<Aside type='warning' header='Important'>
```
Comment (kodster28) [inline products/firewall/src/content/cf-rulesets/custom-rules/custom-firewall/index.md:10]: Same comment as above about the `Aside`.

File: products/firewall/src/content/cf-rulesets/custom-rules/custom-firewall/index.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+This feature is only available for selected customers on an Enterprise plan. Contact your Cloudflare Customer Success Manager if you want to get access.
+
+</Aside>
+
+Custom Firewall rules allow you to control incoming traffic by filtering requests. You can block or challenge incoming requests according to the rules you define.
+
+Like other rules evaluated by Cloudflare's [Ruleset Engine](https://developers.cloudflare.com/firewall/cf-rulesets), Custom Firewall rules have an **expression** and an **action**:
```
Comment (kodster28) [inline products/firewall/src/content/cf-rulesets/custom-rules/custom-firewall/index.md]:
```suggestion
Like other rules evaluated by Cloudflare's [Ruleset Engine](/cf-rulesets), Custom Firewall rules have an **expression** and an **action**:
```

File: products/firewall/src/content/cf-rulesets/custom-rules/manage-dashboard.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+---
+order: 5
+pcx-content-type: how-to
+---
+
+# Manage Custom Rules in the dashboard
+
+<Aside type='warning' header='Important'>
```
Comment (kodster28) [inline products/firewall/src/content/cf-rulesets/custom-rules/manage-dashboard.md:10]: Same comment as above about the `Aside`

File: products/firewall/src/content/cf-rulesets/custom-rules/rate-limiting/index.md [hunk 7]
```diff
@@ -3,44 +3,37 @@ pcx-content-type: concept
...
+---
 
 ## Getting started
 
-To configure Rate Limiting Rules using the Cloudflare dashboard, use the **Custom Rules** tab in the **Firewall** app. For more information, see [Manage Rate Limiting Rules in the dashboard](/cf-rulesets/custom-rules/rate-limiting/manage-dashboard).
+To configure Rate Limiting rules using the Cloudflare dashboard, use the **Custom Rules** tab in the **Firewall** app. For more information, see [Create Rate Limiting rules in the dashboard](/cf-rulesets/custom-rules/rate-limiting/create-dashboard).
 
-You can also configure Rate Limiting Rules using the [Rulesets API](/cf-rulesets/rulesets-api). See [Manage Rate Limiting Rules via API](/cf-rulesets/custom-rules/rate-limiting/manage-api) for more information.
\ No newline at end of file
+You can also configure Rate Limiting rules using the [Rulesets API](/cf-rulesets/rulesets-api). See [Create Rate Limiting rules via API](/cf-rulesets/custom-rules/rate-limiting/create-api) for more information.
```
Comment (kodster28) [inline products/firewall/src/content/cf-rulesets/custom-rules/rate-limiting/index.md:39]: It looks like you're changing the URLs for these examples. Just a reminder — you probably already did — to add redirects for these.

**Review by kodster28 — APPROVED** [review]

File: products/firewall/src/content/cf-rulesets/custom-rules/rate-limiting/index.md (hunk 7 above)
Comment (pedrosousa) [reply products/firewall/src/content/cf-rulesets/custom-rules/rate-limiting/index.md:39]: Will do!

**Pull request merged**